src/main/kotlin/com/terraformation/backend/accelerator/api/GlobalRolesController.kt

```diff
@@ -0,0 +1,75 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.customer.db.UserStore
+import com.terraformation.backend.customer.model.IndividualUser
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/globalRoles")
+@RestController
+class GlobalRolesController(
+    private val userStore: UserStore,
+) {
+  @ApiResponse200
+  @GetMapping("/users")
+  @Operation(summary = "Gets the list of users that have global roles.")
+  fun listGlobalRoles(): GlobalRoleUsersListResponsePayload {
+    requirePermissions { readGlobalRoles() }
+
+    return GlobalRoleUsersListResponsePayload(
+        userStore.fetchWithGlobalRoles().map { UserWithGlobalRolesPayload(it) })
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @PostMapping("/users/{userId}")
```

Comment:
If this controller moves under `/api/v1/users`, then this path could turn into `/api/v1/users/{userId}/globalRoles` which, to me, feels more consistent with how we do paths elsewhere in the API.